Tasks/VsTestV2/task.json

```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 2,
         "Minor": 237,
-        "Patch": 0
+        "Patch": 2
```

Comment:
I wasn't able to update it to version 2.237.1 because of below error:
![Uploading image.png…]()

Reply:
This is expected, since we are leveraging 2.237.1 for the node 20 version of the task. With this update, the node 20 version of the task will jump to 2.237.3

Reply:
This is expected. Since 2.237.1 will be used for Node 20 version of the task. After this PR is merged, 2.237.3 will be used for newer node 20 version of the task.